templates/sast/template.yaml

```diff
@@ -7,7 +7,7 @@ parameters:
   - name: sonarqube_token
     displayName: SonarQube Token
     type: credentials
-  - name: default_source_path
+  - name: default_sonarqube_source_path
```

Comment:
This could be a breaking change, which I think is still ok at this stage, but it would be wise to check if this has been customized for an EPPE or QMMS pipelines. If this parameter was set in the Jenkins dashboard it may lose its value when this is deployed.

Reply:
I'll check this over.